loadflow/loadflow-api/src/main/java/com/powsybl/loadflow/LoadFlow.java

```diff
@@ -6,15 +6,189 @@
+        }
+
+        public LoadFlowResult run(Network network, ComputationManager computationManager, LoadFlowParameters parameters) {
+            return run(network, network.getVariantManager().getWorkingVariantId(), computationManager, parameters);
+        }
+
+        public LoadFlowResult run(Network network, LoadFlowParameters parameters) {
+            return run(network, LocalComputationManager.getDefault(), parameters);
+        }
+
+        public LoadFlowResult run(Network network) {
+            return run(network, LoadFlowParameters.load());
+        }
+
+        @Override
+        public String getName() {
+            return provider.getName();
+        }
+
+        @Override
+        public String getVersion() {
+            return provider.getVersion();
+        }
+    }
+
+    /**
+     * Get a runner for loadflow implementation named {@code name}. In the case of a null {@code name}, default
+     * implementation is used.
+     *
+     * @param name name of the load flow implementation, null if we want to use default one
+     * @return a runner for loadflow implementation named {@code name}
+     */
+    public static Runner find(String name) {
+        return find(name, PROVIDERS_SUPPLIERS.get(), PlatformConfig.defaultConfig());
+    }
+
+    /**
+     * Get a runner for default loadflow implementation.
+     *
+     * @throws PowsyblException in case we cannot find a default implementation
+     * @return a runner for default loadflow implementation
+     */
+    public static Runner find() {
+        return find(null);
+    }
+
+    /**
+     * A variant of {@link LoadFlow#find(String)} intended to be used for unit testing that allow passing
+     * an explicit provider list instead of relying on service loader and an explicit {@link PlatformConfig}
+     * instead of global one.
+     *
+     * @param name name of the load flow implementation, null if we want to use default one
+     * @param providers loadflow provider list
+     * @param platformConfig platform config to look for default loadflow implementation name
+     * @return a runner for loadflow implementation named {@code name}
+     */
+    public static Runner find(String name, List<LoadFlowProvider> providers, PlatformConfig platformConfig) {
+        Objects.requireNonNull(providers);
+        Objects.requireNonNull(platformConfig);
+
+        if (providers.isEmpty()) {
+            throw new PowsyblException("No loadflow providers found");
+        }
+
+        // if no loadflow implementation name is provided through the API we look for information
+        // in platform configuration
+        String loadFlowName = name != null ? name : platformConfig.getOptionalModuleConfig("load-flow")
+                                                                  .flatMap(mc -> mc.getOptionalStringProperty("default"))
+                                                                  .orElse(null);
```

Comment:
Don't forget to write a new page in the configuration section of the website

Reply:
Done